com.playeveryware.eos/Runtime/Core/Utility/FileSystemUtility.cs

```diff
@@ -20,6 +20,9 @@
  * SOFTWARE.
  */
 
+using System.Runtime.CompilerServices;
```

Comment:
Can this be moved under namespace, or does it need to be defined before the InternalsVisibleTo attribute piece?

Reply:
It needs to be defined before the `InternalsVisibleTo` attribute.